pinot-core/src/main/java/com/linkedin/pinot/core/startree/hll/ArrayConverter.java

```diff
@@ -0,0 +1,24 @@
+package com.linkedin.pinot.core.startree.hll;
```

Comment:
Need to add the Apache license header. Run `mvn license:format` and update your branch.